ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/privacy/OnChainPrivacyPrecompiledContract.java

```diff
@@ -323,6 +353,11 @@ private boolean isMemberOfPrivacyGroup(
     return participants;
   }
 
+  private String getRemovedParticipantFromParameter(final Bytes input) {
+    final Bytes mungedParticipants = input.slice(4);
+    return mungedParticipants.slice(0, 32).toBase64String();
```

Comment:
this should not be necessary, as there is only one parameter in that call that has to be 32 bytes long. As the transaction has succeeded I think it should be safe to assume that it really is 32 bytes long. Maybe checking that is OK as well?

Reply:
simplified